components/Dropdown/Dropdown.js

```diff
@@ -2,24 +2,24 @@ class Dropdown {
   constructor(element) {
     
     // Assign this.element to the dropdown element
-    this.element;
+    this.element = document.querySelector('.dropdown'); 
```

Comment:
You can also just assign *this.element* to the *element* variable.
    *this.element = element;*